File: wallet/src/main/java/com/worldventures/wallet/domain/entity/record/SyncRecordsStatus.java
package com.worldventures.wallet.domain.entity.record;

public enum SyncRecordsStatus {
   SUCCESS, FAIL_AFTER_PROVISION;

   public boolean isSuccess() {
      return this == SUCCESS;
   }

   public boolean isFailAfterProvision() {
      return this == FAIL_AFTER_PROVISION;
   }
}

File: wallet/src/main/java/com/worldventures/wallet/domain/entity/record/SyncRecordsStatus.kt
package com.worldventures.wallet.domain.entity.record

enum class SyncRecordsStatus {
   SUCCESS, FAIL_AFTER_PROVISION;

   val isSuccess: Boolean
      get() = this == SUCCESS

   val isFailAfterProvision: Boolean
      get() = this == FAIL_AFTER_PROVISION
}
